packages/core/config/irving.config.default.js

```diff
@@ -6,7 +6,7 @@ import Placeholder from 'components/placeholder';
 import withLoader from 'components/hoc/withLoader';
 
 const irvingDefaultConfig = {
-  packages: {},
+  packages: [],
```

Comment:
Packages don't need to have keys (but they may need to include a package name, in the future)